src/lib/connectors/crm-v2/documents.js

```diff
@@ -2,13 +2,18 @@
 
 const urlJoin = require('url-join');
 const Joi = require('joi');
+const moment = require('moment');
 
 const { serviceRequest } = require('@envage/water-abstraction-helpers');
 const config = require('../../../../config');
 
 const VALID_LICENCE_NUMBER = Joi.string().required().example('01/123/R01');
 const VALID_GUID = Joi.string().guid().required();
 
+const getDocumentUrl = (...tail) => {
+  return urlJoin(config.services.crm_v2, 'document', ...tail);
+};
+
```

Comment:
The code matches the details in the ticket but I think we should re-use the existing documents connector code because we are working with the same entity:

const getDocumentsUrl = (...tail) => {
  return urlJoin(config.services.crm_v2, 'documents', ...tail);
};

This will require a change to the CRM API endpoint to match and will add a comment to that pull request.